LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
+}
+alias tailpglog='tail -f /var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/postgresql.log'
```

Comment:
Much appreciated, as I didn't know those vars were set.  I have updated the tailpglog command appropriately.